extensions/wikia/PortableInfobox/tests/PortableInfoboxParserTagControllerTest.php

```diff
@@ -201,6 +214,127 @@ public function testGetLayoutDataProvider() {
 	}
 
 	/**
+	 * @dataProvider testGetColorDataProvider
+	 */
+	public function testGetColor( $params, $expectedOutput, $text, $templateInvocation, $message ) {
+		$marker = $this->controller->renderInfobox( $text, $params, $this->parser,
+			$this->parser->getPreprocessor()->newCustomFrame( $templateInvocation ) )[ 0 ];
+		$output = $this->controller->replaceMarkers( $marker );
+
+		$this->assertEquals($this->normalizeHTML($expectedOutput), $this->normalizeHTML($output), $message);
+	}
+
+	public function testGetColorDataProvider() {
+		return [
+			[
+				'params' => [ 'accent-color-default' => '#fff' ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "params" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "accent-color-default" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)